modules/terrain/src/lib/helpers/skirt.js

```diff
@@ -72,6 +75,31 @@ function getOutsideEdges(triangles) {
   return outsideEdges;
 }
 
+/**
+ * Get geometry edges that located on a border of the mesh
+ * @param {object} indices - edge indices from quantized mesh data
+ * @param {TypedArray} position - position attribute geometry data
+ * @returns {number[][]} - outside edges data
+ */
+function getOutsideEdgesFromIndices(indices, position) {
+  // Sort skirt indices to create adjacent triangles
```

Comment:
I think `@loaders.gl/math` has some iterators for getting triangles from either indexed or non-indexed geometries.

Reply:
The task is about creating new triangles. We don't want to touch existing geometry because it slows down processing very much. In this particular function I'm converting `edgeIndices` come from `decode-quantized-mesh.js` to another structure to have further processing of "mapbox" and "quantized" meshes unified.